pywinauto/win32functions.py

```diff
@@ -355,7 +777,7 @@ def LoWord(value):
 def WaitGuiThreadIdle(handle):
     """Wait until the thread of the specified handle is ready"""
     process_id = wintypes.DWORD(0)
-    GetWindowThreadProcessId(handle, ctypes.POINTER(wintypes.DWORD)(process_id))
+    GetWindowThreadProcessId(handle, POINTER(wintypes.DWORD)(process_id))

```

Comment:
This call is different than other calls of this function. Maybe it makes sense to align them (using `byref`, for example).

Reply:
Should be fixed now with a299ab6